Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM        python:3.5
+MAINTAINER  Michael B. Klein <michael.klein@northwestern.edu>
+RUN         mkdir -p /var/app /data
+WORKDIR     /var/app
+ADD         . lakesuperior
+WORKDIR     /var/app/lakesuperior
```

Comment:
See comment to line 4.

Reply:
I’m happy to put this anywhere. I was following the AWS and (sometime) Docker convention of using /var/app, but I agree that /usr/local makes sense. I’ll change it.